assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -1343,6 +1343,28 @@ const baseSelectors = {
 		}
 	),
 
+	/**
+	 * Gets the list of internal shareable modules for dashboard sharing.
+	 *
+	 * Returns an Object/map of objects, keyed by slug as same as `getModules`.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(Object|undefined)} Internal shareable modules available on the site; `undefined` if not loaded.
```

Comment:
The documentation for this function says this returns all _internal_ sharable modules, but in fact the function filters out modules that are internal and only returns the non-internal ones.

The behaviour here is correct, but the documentation is wrong—can you please update it? 🙂